build.gradle.kts

```diff
@@ -129,8 +129,8 @@ androidStudioTests {
     val autoDownloadAndRunInHeadless = providers.gradleProperty("autoDownloadAndRunInHeadless").orNull == "true"
     runAndroidStudioInHeadlessMode.set(autoDownloadAndRunInHeadless)
     autoDownloadAndroidStudio.set(autoDownloadAndRunInHeadless)
-    // Electric Eel (2022.1.1) Beta 1
-    testAndroidStudioVersion.set("2022.1.1.11")
+    // Giraffe (2022.3.1.2) Canary 2
+    testAndroidStudioVersion.set("2022.3.1.2")
```

Comment:
❓ Would it make sense to test multiple versions of Android Studio?

Reply:
That would be good, but I think it needs more work, so I would not do it as part of this PR.

Reply:
Opened https://github.com/gradle/gradle-profiler/issues/476.

Reply:
Sounds good.